### ISSUE TITLE
RPM command generation.

### ISSUE THREAD

**Issue by c0lem3** [AA-CubeSat-Team/soci-gnc#33]
If the wheels never get saturated this is not an issue. It becomes an issue when the new desaturation fix is implemented. The RPM command is generated by integrating torques over time. If the wheels get saturated and enter the null space issue, there is no torque that can solve the problem from a control perspective since the error in the body frame is already zero! As such, the wheels are brought down to their nominal value in "open loop". Since this avoids some torque command, there is no torque to integrate. After the wheels are brought to their nominal values 1000, -1000 etc, which works properly, the rpm goes straight back to being close to what it was before the spindown. This means that our entire method of determining the rpm command for the wheels is flawwed. Clearly, this method does not determine output torques based on what the RWA is actually doing, it's just finding rpm commands based on the control torques it integrates. 

TLDR: the commanded rpm will be wrong if you ever change the commanded rpm outright instead of using control torque.
